test/cmdlineTests/keccak_optimization_deploy_code/input.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-v3
+pragma solidity >= 0.0.0;
+
+contract C {
+    constructor() {
+        assembly {
+            mstore(0, 100)
+            // because this is part of deploy code, the keccak will not be evaluated
+            sstore(0, keccak256(0, 32))
```

Comment:
@bshastry In this example, you can see that keccak in constructors are not evaluated.

Reply:
Will take a closer look at the sol test that is showing an increase